src/main/content/META-INF/vault/filter.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <workspaceFilter version="1.0">
 	<filter root="/apps/cq/core/content/nav/tools/jcr-hopper" />
-	<filter root="/apps/jcr-hopper" />
+	<filter root="/apps/jcr-hopper">
+		<exclude pattern="/apps/jcr-hopper/script-builder/samples(/.*)?" />
+	</filter>
```

Comment:
Maybe we could allow each app to contribute its own collection of samples? I.e. we would allow samples to be placed in any `/apps/*/jcr-hooper-samples` dir.
That way, we could also store the built-in samples the same way and it would be possible for libraries to provide samples.

Reply:
I like the idea of giving other apps a place to add scripts. Maybe a different layout though.

And, rather than call them samples, can we call them scripts. The basic sample scripts can go in their own folder. Our Swisscom specific ones can go in another. Other Apps can add their own subfolder. I would suggest the structure look something like this:

```
+ /apps/jcr-hopper/script-builder
|
+-- scripts
   |
   +-- samples
   |   |
   |   +-- sample-script-1
   |   |
   |   +-- sample-script-2
   |
   +-- swisscom
   |
   +-- …
   
```

Reply:
I think, this way, other apps can still overlay the node with 'merge' and we don't have to hunt around the tree (not that that's a huge deal).

Reply:
Also, with that setup, we'd remove the exclude pattern. We'd want the samples in the sample folder to be installed as part of JCR Hopper. We could consider giving it the merge mode but honestly I think we can be clear that third party scripts should go in their own subfolder. That way we make sure that the sample scripts are always up to date.

Reply:
Yes, don’t use merge mode, it’s use is discouraged even by Adobe. The ideal approach would be to exclude `/apps/jcr-hopper/script-builder/scripts` but include the `/apps/jcr-hopper/script-builder/scripts/samples` subfolder, if that’s possible (not sure that it is, though). Otherwise, I’d still prefer the suggestion with `/apps/*/jcr-hooper-scripts`.